config/v1/types_cluster_version.go

```diff
@@ -141,7 +144,8 @@ type UpdateHistory struct {
 	// that is currently being applied will have a null completion time.
 	// Completion time will always be set for entries that are not the current
 	// update (usually to the started time of the next update).
-	CompletionTime *metav1.Time `json:"completionTime"`
+	// +optional
+	CompletionTime *metav1.Time `json:"completionTime,omitempty"`
```

Comment:
same comment about preferring optional and not omitempty

Reply:
it's now nullable